server/views/components/actions-card/_challengesActions.test.ts

```diff
@@ -19,6 +19,10 @@ const templateParams: ActionsCardParams = {
 }
 
 describe('_challengesActions', () => {
+  beforeEach(() => {
```

Comment:
I forgot these beforeEach hooks in my previous PR re: RBAC - adding them whist I was here in the general area ...